forge-gui/res/cardsfolder/upcoming/ghostly_keybearer.txt

```diff
@@ -0,0 +1,8 @@
+Name:Ghostly Keybearer
+ManaCost:3 U
+Types:Creature Spirit
+PT:3/3
+K:Flying
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | Execute$ TrigUnlock | CombatDamage$ True | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, unlock a locked door of up to one target Room you control.
```

Comment:
@Hanmac
I tried to unlock _Copy Enchantment_ Room but it fails from `getLockedRooms` check 🤔

Reply:
what exactly did you do so i can reproduce it?

Reply:
- Cast _Copy Enchantment_ and copy one of the rooms in this PR
- try to use _Ghostly Keybearer_ trigger to unlock
- it seems to fail because it's not `CardSplitType.Split`

Reply:
check removed